### PR TITLE
Relayer: update docs

### PR DESCRIPTION
**Pull request by sczembor**
Updated docs reflecting the new database schema + diagram

Closes: #68 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
    
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary

### CODE REVIEW

File: docs/db.md [hunk 1]
```diff
@@ -4,21 +4,63 @@ This part describes the database schema and interactions used in the relayer.
...
 The relayer interacts with the database using the following functions in the `dal` package:
 
-1. `InsertTx()`: Inserts a new transaction into the database.
-2. `GetTx()`: Retrieves a transaction by its ID.
-3. `GetSignedTxs()`: Retrieves all transactions with a `signed` status.
-4. `GetBroadcastedTxs()`: Retrives all transactions with a `broadcasted` status.
-5. `UpdateTxStatus()`: Updates the status of a transaction.
+### IKA Sign Requests
+
+1. `InsertIkaSignRequest(request IkaSignRequest)`: Inserts a new IKA sign request into the `ika_sign_requests` table.
```
Comment (robert-zaremba) [inline docs/db.md:48]: I  think we don't need to list all functions. Just list function "categories" (Ika, Bitcoin.... And where they are).

**Review by robert-zaremba — APPROVED** [review]

File: docs/db.md [hunk 2]
```diff
@@ -4,21 +4,63 @@ This part describes the database schema and interactions used in the relayer.
 
 ## Schema
 
-The relayer uses a SQLite database to store and manage transaction data. The database has a single table named `transactions` with the following schema:
+The relayer uses a SQLite database to store and manage transaction data. The database has three tables:
```
Comment (sczembor) [inline docs/db.md:7]:
```suggestion
The relayer uses a SQL database to store and manage transaction data. The database has three tables:
```

File: docs/db.md (hunk 1 above)
Comment (vuvoth) [reply docs/db.md:48]: And update them in change log. I mean basic function 👍🏼

File: README.md [hunk 3]
```diff
@@ -38,7 +38,9 @@ Mock -> WIP -> alpha -> beta -> production
 
 ## Documentation
 
-1. [Database interacitons](docs/db.md)
+1. Database
```
Comment (vuvoth) [inline README.md:41]: Not relate to this line. 
But we should update to "CometBFT blocks to update the Native Light Client in the Ika network"
Same for "Bitcoin blocks..."

**Review by vuvoth — APPROVED** [review]

File: docs/db.md (hunk 1 above)
Comment (sczembor) [reply docs/db.md]: Done

**Review by vuvoth — APPROVED** [review]
LGTM!

**Pull request merged**